pkg/controller/pod.go

```diff
@@ -570,7 +570,10 @@ func (c *Controller) handleDeletePod(pod *v1.Pod) error {
 	c.podKeyMutex.Lock(key)
 	defer c.podKeyMutex.Unlock(key)
 
-	p, _ := c.podsLister.Pods(pod.Namespace).Get(pod.Name)
+	p, err := c.podsLister.Pods(pod.Namespace).Get(pod.Name)
+	if err != nil {
+		return err
+	}
```

Comment:
This causes the e2e testing failure. We should ignore error or check whether the error is `Pod not found` since the controller is handling Pod deletion event.

Reply:
ok, ignore error